examples/functions-with-rest-parameter/functions_with_rest_parameter.bal

```diff
@@ -1,7 +1,7 @@
 import ballerina/io;
 
-// This function takes one required parameter and one rest parameter of type 
-// string. A function can have only one rest parameter.
+// This function takes one required parameter, one defaultable parameter and
+// one rest parameter of type string. A function can have only one rest parameter.
```

Comment:
Shall we change 'rest' to 'REST'. Apply elsewhere if applicable.

Reply:
This is rest parameter nothing related to REST(REpresentational State Transfer). So kept as it is.